.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+  - nodejs_version: 8
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install -g npm # Force using the latest npm to get dedupe during install
+  - set PATH=%APPDATA%\npm;%PATH%
+  - npm install --force --ignore-scripts
```

Comment:
Sorry for all the questions.  Why --force and --ignore-scripts?

Reply:
Just copied the one we use in googleapis/nodejs-* projects. To answer your questions.
(a) as the comment states - `Force using the latest npm to get dedupe during install`. Not sure if it's actual, I'm OK to remove.
(b) I'm pretty sure some scripts work only on Linux and we don't want to fail if any of those fail.

Reply:
Wouldn't those scripts fail an `npm install` on Windows though?

Reply:
Not sure about `--force`: as I said, I just copied it from another place :) We can try removing it and see what happens.

Reply:
Lets start without them, and add them back if needed.  Lets keep it as close as possible to the actual developer workflow